media-sound/supercollider/supercollider-3.11.0.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake readme.gentoo-r1 xdg-utils
+
+DESCRIPTION="An environment and a programming language for real time audio synthesis."
+HOMEPAGE="https://supercollider.github.io/"
+SRC_URI="https://github.com/supercollider/supercollider/releases/download/Version-${PV}/SuperCollider-${PV}-Source.tar.bz2"
+
+LICENSE="GPL-2 gpl3? ( GPL-3 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_sse cpu_flags_x86_sse2 debug emacs +fftw gedit +gpl3 jack qt5 server +sndfile static-libs vim X zeroconf"
+
+REQUIRED_USE="qt5? ( X )"
+
+BDEPEND="
+	virtual/pkgconfig
+	qt5? ( dev-qt/linguist-tools:5 )
+"
+RDEPEND="
+	dev-cpp/yaml-cpp:=
+	>=dev-libs/boost-1.70.0:=
+	media-libs/alsa-lib
+	sys-libs/readline:0=
+	virtual/libudev:=
+	fftw? ( sci-libs/fftw:3.0= )
+	jack? ( virtual/jack )
+	!jack? ( media-libs/portaudio )
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtopengl:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtsql:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwebchannel:5
+		dev-qt/qtwebengine:5[widgets]
+		dev-qt/qtwebsockets:5
+		dev-qt/qtwidgets:5
+	)
+	server? ( !app-admin/supernova )
+	sndfile? ( media-libs/libsndfile )
+	X? (
+		x11-libs/libX11
+		x11-libs/libXt
+	)
+	zeroconf? ( net-dns/avahi )
+"
+DEPEND="${RDEPEND}
+	dev-libs/icu
```

Comment:
no linking to icu? that would be a rare sight.

Reply:
Seems like this is an old dep that is no longer required. Dropped.